plugins/generator-1.16.5/forge-1.16.5/procedures/get_command_parameter_blockstate.java.ftl

```diff
@@ -0,0 +1 @@
+(BlockStateArgument.getBlockState(arguments, "${field$param}").getState().getBlock())
```

Comment:
Aren't you missing `/*@BlockState*/` here? Did you test 1.16.5?

Reply:
No, it's not missing as it is not required to work as it should and yes, I also tested in 1.16.5.

Reply:
```suggestion
/*@BlockState*/(BlockStateArgument.getBlockState(arguments, "${field$param}").getState().getBlock())
```